<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cripty</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Current files in vault&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="currentFilesComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>431</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFileButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptFileButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt File</string>
    </property>
   </widget>
   <widget class="QLabel" name="vaultNameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Current Vault Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewVault"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVault">
    <property name="title">
     <string>Vault</string>
    </property>
    <addaction name="actionCloseVault"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_vault"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCripty_Help"/>
    <addaction name="actionAbout"/>
    <addaction name="actionGitHub_repo"/>
    <addaction name="separator"/>
    <addaction name="actionReport_Issue"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAdd_Files"/>
    <addaction name="actionExport_Vault"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVault"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCloseVault">
   <property name="icon">
    <iconset theme="dialog-password"/>
   </property>
   <property name="text">
    <string>Lock Vault</string>
   </property>
  </action>
  <action name="actionRemove_vault">
   <property name="text">
    <string>Remove vault</string>
   </property>
  </action>
  <action name="actionAdd_Files">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionExport_Vault">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Export Vault</string>
   </property>
  </action>
  <action name="actionCripty_Help">
   <property name="icon">
    <iconset theme="help-browser"/>
   </property>
   <property name="text">
    <string>Cripty Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionGitHub_repo">
   <property name="text">
    <string>GitHub Repo</string>
   </property>
  </action>
  <action name="actionReport_Issue">
   <property name="icon">
    <iconset theme="dialog-error"/>
   </property>
   <property name="text">
    <string>Report Issue</string>
   </property>
  </action>
  <action name="actionNewVault">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderNew"/>
   </property>
   <property name="text">
    <string>New Vault</string>
   </property>
   <property name="toolTip">
    <string>Create a new Vault</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
